coral-datetime/src/scripts/DateTime.js

```diff
@@ -280,20 +296,49 @@ class DateTime {
    @see https://momentjs.com/docs/#/manipulating/subtract/
    */
   subtract(value, type) {
-    if (type.indexOf('month') === 0) {
-      this._date.setMonth(this._date.getMonth() - value);
-    } else if (type.indexOf('day') === 0) {
-      this._date.setDate(this._date.getDate() - value);
+    let multiplier = 1;
+    switch (type) {
+      case 'year':
+      case 'years':
+        multiplier = 12;
+      case 'month':
+      case 'months':
+        const dayOfMonth = this._date.getDate();
+        this._date.setMonth(this._date.getMonth() - multiplier * value);
+        if (this._date.getDate() != dayOfMonth) {
+          this._date.setDate(0);
+        }
+        break;
+      case 'week':
+      case 'weeks':
+        multiplier = 7;
+      case 'day':
+      case 'days':
+        this._date.setDate(this._date.getDate() - multiplier * value);
+        break;
     }
 
     return this;
   }
 
+
+  /**
+   @see https://momentjs.com/docs/#/displaying/days-in-month/
+  */
+  daysInMonth() {
```

Comment:
Couldn't find its usage, if unused code please remove it.

Reply:
https://github.com/adobe/coral-spectrum/pull/148/files#diff-1814d617222a9f2c5f83f21f9b4e4877e0dbbbe630a731975cab53e5bbedfabaR1000